themes/scss/map/dashboard-info.scss

```diff
@@ -73,7 +70,8 @@
   margin: 0 auto;
   border: 2px solid #FFF;
   border-radius: 50%;
-  background: url('../img/dashboard-show.svg') no-repeat center center;
+  color: #FFF;
+  font-size: $sFontSize-medium;
```

Comment:
Name of variable `sFontSize-medium` should be written in all lowercase letters with hyphens instead of underscores